.travis/get-or-build-pill.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+set -euo pipefail
+# set -x
+
+# XX use -s instead of hash pill
+HASH=$(git -C .. log -1 HEAD --format=%H -- sys/)
+export PILL_NAME="git-${HASH:0:10}"
+
+if [ ! ${PILL_FORCE:-} ]; then
+  wget https://bootstrap.urbit.org/$PILL_NAME.pill -O urbit.pill && exit 0
+fi
+
+# if wget failed
+
+if [ ${TRAVIS_COMMIT:-} ] && [ $TRAVIS_COMMIT != $HASH ]; then
+  echo Directory sys/ not modified in commit $TRAVIS_COMMIT
+  echo FIXME ignoring, as current sys/ commits are unlikely to contain the pill-build code
+  echo
+#   echo For auto-build please tag and push $HASH
+#   exit 1
+fi
+
+mkdir prev
+{
+  echo Pilling: trying pinned fakezod
+  wget -i pin-parent-pill-pier.url -O - | tar xvz -C prev/ &&
+  echo Downloaded prev/zod
+} || {
+  echo Pilling: Parent-pill pier not available, trying preceding pill commit
+  HASH2=$(git -C .. log -2 $HASH --format=%H -- sys/ | tail -1)
```

Comment:
This is the hash of the previous commit. We should probably name this variable accordingly, like PREV_HASH.